src/openzaak/components/zaken/openapi.yaml

```diff
@@ -24,13 +24,9 @@ info:
     \n**Main resource**\n\n`zaak`\n\n\n\n**Kenmerken**\n\n* `bronorganisatie`: Het\
     \ RSIN van de Niet-natuurlijk persoon zijnde de organisatie die de zaak heeft\
     \ gecreeerd. Dit moet een geldig RSIN zijn van 9 nummers en voldoen aan https://nl.wikipedia.org/wiki/Burgerservicenummer#11-proef\n\
-    * `zaaktype`: URL-referentie naar het ZAAKTYPE (in de Catalogi API) in de CATALOGUS\
-    \ waar deze voorkomt\n* `vertrouwelijkheidaanduiding`: Aanduiding van de mate\
-    \ waarin het zaakdossier van de ZAAK voor de openbaarheid bestemd is.\n\n**Resources\
-    \ en acties**\n- `zaak`: create, update, destroy\n- `status`: create\n- `zaakobject`:\
-    \ create\n- `zaakinformatieobject`: create\n- `zaakeigenschap`: create\n- `klantcontact`:\
-    \ create\n- `rol`: create\n- `resultaat`: create, update, destroy\n- `zaakbesluit`:\
-    \ create\n\n\n**Handige links**\n\n* [Documentatie](https://zaakgerichtwerken.vng.cloud/standaard)\n\
+    * `zaaktype`: URL-referentie naar het ZAAKTYPE (in de Catalogi API).\n* `vertrouwelijkheidaanduiding`:\
+    \ Aanduiding van de mate waarin het zaakdossier van de ZAAK voor de openbaarheid\
+    \ bestemd is.\n\n**Resources en acties**\n\n\n**Handige links**\n\n* [Documentatie](https://zaakgerichtwerken.vng.cloud/standaard)\n\
```

Comment:
there's a part missing now of the notifications doc, because some import doesn't happen in time. I think it could be fixed by importing the viewsets in the app config `ready()` method

Reply:
Created https://github.com/open-zaak/open-zaak/issues/299 for this.